scripts/fastertransformer/faster_transformer_install.sh

```diff
@@ -1,3 +1,4 @@
+#! /bin/bash
```

Comment:
Looking at this, we should be able to have multiple folders. One for each DSM and we can have multiple builds each using a different DSM in each folder. This would allow us to have one convert image.

Reply:
Are you suggesting that we end up with a single image with a file structure like:

\/workspace/FasterTransformer/build/70/bin
\/workspace/FasterTransformer/build/75/bin
...

Reply:
If this works, then yes. 

if it doesn't, we can have a different clone , something like
/workspace/FasterTransformer70/build
/workspace/FasterTransformer75/build

Reply:
If this works, then yes. 

if it doesn't, we can have a different clone , something like
/workspace/FasterTransformer70/build
/workspace/FasterTransformer75/build

Reply:
We'll see how much this bloats the image.

Napkin math is putting it at ~50GB but will know in about an hour when it's done building.

Reply:
Reading back through the FasterTransformer docs it looks like we may be overengineering this solution.

The [docs](https://github.com/NVIDIA/FasterTransformer/blob/main/docs/t5_guide.md#build-the-project) suggest that we can build for multiple DSM with no concerns.

"By default, -DSM is set by 70, 75, 80 and 86. When users set more kinds of -DSM, it requires longer time to compile. So, we suggest setting the -DSM for the device you use only. Here, we use xx as an example due to convenience."

I've reached out to the repo for more #https://github.com/NVIDIA/FasterTransformer/issues/507

Reply:
Done. There is now one single Convert image built for all supported GPU architectures.